app/helpers/__init__.py

```diff
@@ -1,10 +1,15 @@
 from .address_lookup_api_helper import get_address_lookup_api_auth_token
 from .header_helpers import get_span_and_trace
-from .template_helpers import get_census_base_url
+from .template_helpers import context_options
 from .url_safe_serializer import url_safe_serializer
 
+
+def get_base_url(schema_theme: str, language_code: str) -> str:
+    return context_options(schema_theme, language_code).base_url
```

Comment:
It's a bit odd that this method accepts a `language_code` parameter but only ever gets passed `None`. 

I think we should instead have a helper method for getting the survey config, that does something like this:

```python
def get_survey_config():
    theme = cookie_session.get("theme", current_app.config["SURVEY_TYPE"])
    language = get_locale().language
    return context_options(theme, language)
```

This could then be called directly from the sign out page and the `render_template` code could be refactored to use it.

Reply:
Done